src/fluree/db/flake/flake_db.cljc

```diff
@@ -560,9 +561,10 @@
                          (<? (vocab/load-schema indexed-db (:preds root-map)))
                          indexed-db)
            index-t     (:t indexed-db*)]
-       (if (= commit-t index-t)
-         indexed-db*
-         (<? (load-novelty commit-catalog indexed-db* index-t commit-jsonld)))))))
+       (<? (shacl/hydrate-shape-cache!
```

Comment:
I think you should add another `let` binding here instead of this deeply nested async call.